CMakeLists.txt

```diff
@@ -143,6 +143,10 @@ add_definitions(${LLVM_DEFINITIONS})
 # Third party deps
 include(cmake/ExternalProjects.cmake)
 
+# Faabric
+add_subdirectory(faabric)
+include_directories(${CMAKE_CURRENT_LIST_DIR}/faabric/include)
```

Comment:
This is exactly what we were just talking about with Jakub :see_no_evil:. We should add this as a public target include to the faabric target, but I'll fix _all_ these CMake issues in one go.